### PR TITLE
DO NOT MERGE - I AM USING IT FOR TESTs

### PR DESCRIPTION
**Pull request by camilamacedo86**
DO NOT MERGE

### CODE REVIEW

File: scripts/common/rook.sh [hunk 1]
```diff
@@ -198,10 +198,27 @@ function rook_ceph_to_sc_migration() {
...
+                logWarn "Rook object store was successfully migrate but not PVCs. Therefore, rook can not be removed"
+            fi
+            if [ "$DID_MIGRATE_ROOK_PVCS" == "1" ] && [ "$DID_MIGRATE_ROOK_OBJECT_STORE" == "1" ]; then
+               logStep "Removing Rook"
+               report_addon_start "rook-ceph-removal" "v1"
+               remove_rook_ceph
+               report_addon_success "rook-ceph-removal" "v1"
+            fi
+        fi
+        if [ -z "$MINIO_VERSION" ]; then
```
Comment (emosbaugh) [inline scripts/common/rook.sh:215]: I dont think we should remove the object store data when it was not migrated. losing data is a worst case scenario

**Review by emosbaugh — CHANGES_REQUESTED** [review]
I am not sure what this pull request is solving

File: scripts/common/rook.sh (hunk 1 above)
Comment (camilamacedo86) [reply scripts/common/rook.sh:215]: Have MINIO is not mandatory. What we are doing here is calling `remove_rook_ceph` when Minio is not installed.

Comment (camilamacedo86) [reply scripts/common/rook.sh:215]: Minio is not mandatory. What should we do when MINIO is not selected?

**Pull request closed without merging**